cmd/kyma/apply/function/function.go

```diff
@@ -0,0 +1,281 @@
+package function
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/kyma-incubator/hydroform/function/pkg/client"
+	"github.com/kyma-incubator/hydroform/function/pkg/manager"
+	"github.com/kyma-incubator/hydroform/function/pkg/operator"
+	resources "github.com/kyma-incubator/hydroform/function/pkg/resources/unstructured"
+	"github.com/kyma-incubator/hydroform/function/pkg/workspace"
+	"github.com/kyma-project/cli/internal/cli"
+	"github.com/kyma-project/cli/internal/kube"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+type Operators = map[operator.Operator][]operator.Operator
```

Comment:
```suggestion
type Operators map[operator.Operator][]operator.Operator
```

Reply:
Why? This is only an alias to the string, to clarify the code

Reply:
Type aliases should not be used in regular code. They were introduced in Golang to do codebase refactoring as temporary in between for old/new types migration.

See: https://yourbasic.org/golang/type-alias/ 
And here detailed explanation: https://talks.golang.org/2016/refactor.article